.travis/custom_release.sh

```diff
@@ -4,10 +4,12 @@ NODE_ENV=production npm run build
 
 if [ "${TRAVIS_BRANCH}" = "master" ]; then
     .travis/release.sh "ci-beta"
+    .travis/release.sh "ci-beta"
+    
+elif [ "${TRAVIS_BRANCH}" = "stable-master" ]; then
+    .travis/release.sh "qa-stable"
+    .travis/release.sh "ci-stable"
 
-    .travis/promote.sh "qa-beta"
-    .travis/promote.sh "ci-stable"
-    .travis/promote.sh "qa-stable"
```

Comment:
Please don't change this as what you have is not what we want to do.  Also, regardless of the promote vs release thing, it's just not what we want because a) it changes where we want things to auto-deploy and b) has ci-beta twice.

Reply:
But you need to differentiate if you are running things for beta or stable otherwise your beta release will point to stable chrome and that might cause a lot of problems.